charts/ssi-credential-issuer/templates/job-issuer-migrations.yaml

```diff
@@ -66,14 +66,59 @@ spec:
           - name: "SEEDING__TESTDATAENVIRONMENTS__0"
             value: "{{ .Values.migrations.seeding.testDataEnvironments }}"
           - name: "SEEDING__DATAPATHS__0"
-            value: "{{ .Values.migrations.seeding.testDataPaths }}"
+            value: "Seeder/Data"
+          {{- if (.Values.migrations.seeding.testData.useDefaultTestData) }}
+          - name: "SEEDING__DATAPATHS__1"
+            value: "Seeder/Data/import/default"
+          - name: "SEEDING__TESTDATAENVIRONMENTS__1"
+            value: "test"
+          {{- end }}           
+          {{- if and (.Values.migrations.seeding.testData.configMap) (.Values.migrations.seeding.testData.filename) }}
+          {{- if (.Values.migrations.seeding.testData.useDefaultTestData) }}
+          - name: "SEEDING__DATAPATHS__2"
+            value: "Seeder/Data/import/custom"
+          - name: "SEEDING__TESTDATAENVIRONMENTS__2"
+            value: "{{ .Values.migrations.seeding.testData.filename }}"
+          {{ else }}
+          - name: "SEEDING__DATAPATHS__1"
+            value: "Seeder/Data/import/custom"
+          - name: "SEEDING__TESTDATAENVIRONMENTS__1"
+            value: "{{ .Values.migrations.seeding.testData.filename }}"
+          {{- end }}
```

Comment:
I'm not certain about this else statement, is this needed and does it make sense?

Reply:
I've added it for the scenario if one disables the default test data and adds a ConfigMap, in this scenario the array would have index 0 and 2 filled, I think this would result in an error in the backend. therefor I've set it to 1 that index 0 and 1 are filled.

Reply:
ok, once https://github.com/eclipse-tractusx/ssi-credential-issuer/pull/241#discussion_r1773443549 is implemented, I'll test and come back to you